src/ingestion/ec2_describe_instances.py

```diff
@@ -107,31 +107,34 @@ def get_data_worker(account):
     ec2_session = get_aws_client(account)
     if ec2_session:
         instances = []
-        ec2_regions = [region['RegionName'] for region in boto3.client('ec2').describe_regions()['Regions']]
+        ec2_regions = [region['RegionName'] for region in ec2_session.client('ec2').describe_regions()['Regions']]
         for region in ec2_regions:
             ec2_client = ec2_session.client('ec2', region_name=region)
             paginator = ec2_client.get_paginator('describe_instances')
             page_iterator = paginator.paginate()
             for page in page_iterator:
                 for instance_array in page['Reservations']:
                     instances.extend(instance_array['Instances'])
-        instance_list = [json.dumps({**instance, "AccountId": account}, default=str) for instance in instances]
-        if len(instances):
-            sf_client = get_snowflake_client()
-            query = LOAD_INSTANCE_LIST_QUERY.format(
-                snapshotclock=datetime.datetime.utcnow().isoformat(),
-                format_string=", ".join(["(%s)"] * len(instance_list)))
-            sf_client.cursor().execute(query, instance_list)
-
+        instance_list = [json.dumps({**instance,"AccountId":account}, default=str) for instance in instances]
+        return instance_list
 
 def get_data(accounts_list):
     start = datetime.datetime.now()
-    with Pool(processes=4) as pool:
-        pool.map(get_data_worker, accounts_list)
+    instance_list_list = list(filter(None,Pool(4).map(get_data_worker, accounts_list)))
+    instance_list = []
+    for sub_list in instance_list_list:
+        instance_list.extend(sub_list)
+    if len(instance_list):
+        sf_client = get_snowflake_client()
+        instance_groups = [instance_list[i:i+15000] for i in range(0, len(instance_list), 15000)]
```

Comment:
please use `utils.groups_of` for this, unless there's something broken about it, in which case let's fix it

Reply:
the broken part is the part where I don't know how to use utils.groups_of in random places